requirements.txt

```diff
@@ -7,4 +7,8 @@ litellm==1.42.1
 Pillow>=10.1
 beautifulsoup4>=4.12
 fastapi
-uvicorn
\ No newline at end of file
+uvicorn
+chromadb
+llama-index-core
+llama-index-vector-stores-chroma
+llama-index-embeddings-openai
```

Comment:
Could you add the versions here? Since these packages have new versions every other day.